cli/src/semgrep/meta.py

```diff
@@ -593,15 +621,39 @@ def _fetch_branch_get_merge_base(branch_name: str, head_sha: str) -> str:
 
     @property
     def repo_name(self) -> str:
-        return os.getenv("CI_PROJECT_PATH", "[unknown]")
+        project_path = os.getenv("CI_PROJECT_PATH")
+        semgrep_repo_name = os.getenv("SEMGREP_REPO_NAME")
+
+        # see NOTE: no-super
+        if semgrep_repo_name:
+            return semgrep_repo_name
+        if project_path:
+            return project_path
+        return super().repo_name
```

Comment:
Why see NOTE: no-super when you call super?

Reply:
super, no, I also see NOTE: no-super but want no NOTE: no-super

Reply:
I suppose this note is meant to be moreso describing why I don't call `super()` _first_. I isolate the part of the `super()` call that I want, which is the `SEMGREP_REPO_NAME`, but because the `super()` call is otherwise side effecting, I avoid calling that first, prior to looking at the actual `CI_PROJECT_PATH` env var.

I can be more explicit about that.